hilfe/handbuch/probleme/index.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: layouts/hilfe/single.njk
+title: Probleme
+weight: 6
+icon: exclamation-triangle
+---
+
+    **"Irren ist menschlich" - und die Fülle gesetzlicher Bestimmungen enorm.**
+
+So gern wir auch auf Anhieb den perfekten
+Dienstplan erstellen würden, schnell ist gegen **Arbeitsrichtlinien oder
+Vertragsvereinbarungen** verstoßen - früh genug entdeckt ein Aufwendiges, doch im
+schlimmsten Fall ein kostspieliges Irren. Um Fehler dieser Art zu vermeiden,
+bevor Sie überhaupt entstehen, macht Pentacode Sie auf Probleme aufmerksam, die
+während der Dienstplanung aufkommen. So können Sie sich sicher sein, dass Sie **stets im Recht handeln** und vor einer **Prüfung nichts zu befürchten** haben.
+
+## Übersicht nach Monaten
+
+Klicken Sie auf den Reiter {%button label="Probleme",
+icon="exclamation-triangle"%} um in die Problemübersicht zu gelangen. Sie werden
+anschließend eine **chronologische Auflistung** der von Pentacode erkannten
+Problemmeldungen erhalten. Dabei werden die neusten Problemberichte ganz oben
+angezeigt, beispielweise von einem gerade erstellten Dienstplan. Sie können
+entweder durch scrollen in abgeschlossene Monate gelangen oder den gewünschten Monat
+direkt in der linken unteren Ecke auswählen. Eine rot umrahmte Zahl zeigt
+Ihnen dabei wie viele Probleme im betroffenen Monat aufgekommen sind. 
+
+{% figure caption="Wählen Sie den gewünschten Monat in der Liste aus." %}
+<img src="probleme-übersicht.png"/>
+{%endfigure%}
+
+## Nach Art des Problems filtern
+
+Sie können bestimmen welche Problemberichte Ihnen angezeigt werden, indem Sie
+verschiedene "Problemarten" an- und abwählen. Standardmäßig werden Ihnen alle
+Probleme angezeigt, um eine Art von Problembericht abzuwählen, klicken Sie diesen
+in der oberen rechten Ecke an. 
+
+{% figure caption="Da alle anderen Problemberichte abgewählt sind werden hier
+nur *nicht angetretene Schichten* angezeigt." %}
+<img src="probleme-filtern.png"/>
+{%endfigure%}
```

Comment:
Zeilenumbrüche sind innerhalb von `caption="..."` nicht erlaubt, deshalb gibt es hier einen Syntaxfehler. Wir wollen jetzt aber sowieso von der Nunjuck-Syntax weg, weil sie nicht wirklich Vorteile gegenüber einfachem HTML bietet und (wie hier) nur Probleme verursacht. Also in bitte Abbildungen in Zukunft wie folgt schreiben:

```html
<figure>
    <img src="probleme-filtern.png"/>
    <figcaption>Da alle anderen Problemberichte abgewählt sind werden hier
nur nicht angetretene Schichten angezeigt.</figcaption>
</figure>
```

Dann sollte es auch keine Probleme mit den Zeilenumbrüchen geben. Markdown syntax (zb. \*\*fett\*\*) kannst du im Untertitel übrigens nicht verwenden (weder mit HTML noch Nunjucks), also die \* bitte weglassen.